common/presenters/move-to-card-component.js

```diff
@@ -29,6 +30,20 @@ function moveToCardComponent({
 
     const moveMetaItems = []
 
+    if (showIdentifierNumber && profile && profile.person && move.to_location) {
```

Comment:
I'm wondering if we need the `move.to_location` check here? What's the scenario we're trying to prevent with this check?

Reply:
Could also shorten `profile && profile.person` to `profile?.person`.
```suggestion
    if (showIdentifierNumber && profile?.person && move.to_location) {
```